lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -251,6 +251,50 @@ class ConvertTorchConstantIntOp
 };
 } // namespace
 
+namespace {
+class ConvertAtenFloatScalarOp : public OpConversionPattern<AtenFloatScalarOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(AtenFloatScalarOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    Type resultType =
+        this->getTypeConverter()->convertType(op->getResult(0).getType());
+    Value result =
+        convertScalarToDtype(rewriter, op.getLoc(), adaptor.getA(), resultType);
+    rewriter.replaceOp(op, result);
+    return success();
+  }
+};
+} // namespace
+
+namespace {
+class ConvertAtenAddOp : public OpConversionPattern<AtenAddOp> {
```

Comment:
Can you also add an e2e test for this op?

Reply:
Hm, yeah I can see that, since the inputs are supposed to be `Scalar`. I'll see if I can think of something